zk/realtime/realtimeapi/api_node.go

```diff
@@ -0,0 +1,17 @@
+package realtimeapi
+
+import (
+	"context"
+	"fmt"
+)
+
+// Returns the status on whether the RT feature is enabled (when tag provided)
+func (api *RealtimeAPIImpl) RealtimeEnabled(ctx context.Context, tag RealtimeTag) (bool, error) {
+	if api.cacheDB == nil || !api.cacheDB.ReadyFlag.Load() {
```

Comment:
We can simplify the entire RPC handler here by just returning the boolean expression result
```suggestion
	return (api.cacheDB != nil && api.cachDB.ReadyFlag.Load)
```